recipes-wigwag/mbed-edge-core/files/rpi3/arm_update_activate.sh

```diff
@@ -19,12 +19,12 @@ VALUE=$(cp $HEADER /userdata/extended/header.bin)
 mkdir -p /userdata/.logs-before-upgrade
 cp -R /wigwag/log/* /userdata/.logs-before-upgrade/
 
-killall maestro
-/etc/init.d/maestro.sh start
+systemctl stop maestro
+systemctl start maestro
 
 mv $FIRMWARE /upgrades/firmware.tar.gz
 tar -xzf /upgrades/firmware.tar.gz -C /upgrades/
-/etc/init.d/deviceOS-watchdog start
+systemctl start deviceos-wd
```

Comment:
Why are services started if a reboot is initiated?

Reply:
It's insurance. This starts the watchdog service. It is a fail safe option in case reboot commands fails or get stuck, watchdog will definitely power cycle the processor. In our experience with Allwinner chips we have observed that a lot.